public/500.html

```diff
@@ -6,6 +6,6 @@
 <body>
   <h1>Lobsters HTTP 500 status: server error</h1>
   <p>The page you requested cannot be displayed.</p>
-  <p>The site operators have been notified of this error.  You're welcome to and encouraged to also report it in <a href="irc://irc.freenode.net/lobsters">IRC channel</a>.</p>
+  <p>The site operators have been notified of this error.  You're welcome to and encouraged to also report it in the <a href="irc://irc.freenode.net/lobsters">IRC channel</a> (also accessible via <a href="https://webchat.freenode.net/#lobsters">web<a>).</p>
```

Comment:
Will you say "webchat" rather than "web" in the anchor text?  I like being a titch more precise here given the circumstances (site failure) and attendant stress that might cause a reader.